index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+  <head>
+    <link rel="stylesheet" href="style.css">
+    <meta  charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <title>Form-basic</title>
+
+  </head>
+
+  <body>
+    <!-- Top level container -->
+    <div class="container">
+
+      <!-- PAGE HEADER -->
+      <DIV  class="header" >
```

Comment:
Please endeavour to use HTML tags correctly. Tags in HTML 5 are written in lowercare